### ISSUE TITLE
My code

### ISSUE THREAD

**Issue by NotFluyd** [glixzzy/blooket-hack#1212]
This is going to be DMCA striked. You stole the code I sent to you a few months ago. I'm not credited not did you ask for permission.

**Comment by Rentuh** [comment]
Wait WHAT

**Comment by GooseterV** [comment]
^^

**Comment by UndercoverGoose** [comment]
I would love to see the DMCA strike, can't wait!

**Comment by Minesraft2** [comment]
sounds pog

**Comment by ThatFruedDued** [comment]
yeah this DMCA strike is going to be the best entertainment content since baby shark dance

**Comment by suckmytitties** [comment]
bruh

**Comment by Rentuh** [comment]
> yeah this DMCA strike is going to be the best entertainment content since baby shark dance

Well this is my first day on this acc and thats gonna be exciting
